components/products/productCard.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Product } from "./product";
+import Link from "next/link";
+
+const PRODUCT_IMAGES = [
+  "/images/product1.jpg",
+  "/images/product2.jpg",
+  "/images/product3.jpg",
+];
+
+interface ProductCardProps {
+  product: Product;
```

Comment:
[RC]
Saw this Product interface is coming from `product.tsx` file. In React codebase, we use .tsx to show that it is a file that returns renderable react component with jsx syntax. Since there isn't any react component in it, let's rename that file to `product.ts` 

Furthermore, may I suggest that we put that file under `root/entities` folder? Product interface (or type for that matter) is an entity that is related to the business(or domain). It would be nice to organize entities outside of app components.